docs/docs/etl-pipeline-tutorial/3-data-quality.md

```diff
@@ -51,16 +51,16 @@ The asset check is using the same `DuckDBResource` resource we defined for the a
 
 With the Dagster UI, you can now see that an asset check is associated with the `joined_data` asset.
```

Comment:
```suggestion
In the Dagster UI, you can now see that an asset check is associated with the `joined_data` asset:
```

Reply:
When was the `joined_data` asset introduced? Maybe part of the transform step is missing?